guides/introduction/quick_start_guide.md

```diff
@@ -22,28 +22,18 @@ Now that your MeiliSearch server is up and running, you should be able to commun
 
 Communication to the server is done through a [RESTful API](/references/README.md) or one of our [SDKs](/resources/sdks.md).
 
-## Create your Index
-
-In MeiliSearch, the information is subdivided into indexes. Each [index](/guides/main_concepts/indexes.md) contains a data structure and associated documents.
-Indexes can be comparable to SQL tables. Since MeiliSearch is <clientGlossary word="schemaless"/>, there's no need to define any attributes or data type when creating a table.
-
-In order to be able to store your documents in an index, it is required you create one first.
-
-<code-samples id="getting_started_create_index_md" />
-
-[API references](/references/indexes.md)
-
 ## Add Documents
 
-Once the index has been created, the next step is to fill it with [documents](/guides/main_concepts/documents.md). These documents will be used and returned when search queries will be performed on MeiliSearch.
+To add documents to MeiliSearch you must provide [documents](/guides/main_concepts/documents.md) in the form of `JSON objects`, as well as a name for the [index](/guides/main_concepts/indexes.md) where they will be stored. If you try to add documents to an index that doesn't already exist, an index with the given name is automatically created. It is also possible to create indexes manually using the [create index endpoint](/references/indexes.md#create-an-index).
+
+Documents are the basic building blocks of your database. Whenever you perform a search query on an index, MeiliSearch examines the data contained in your documents and returns the ones that match your query, in order of [relevancy](/guides/main_concepts/relevancy.html). 
```

Comment:
```suggestion
To add [documents](/guides/main_concepts/documents.md) to MeiliSearch, you must provide not only the documents themselves, but also a name for the [index](/guides/main_concepts/indexes.md) where they will be stored. Don't worry if you don't have an index yet—when you attempt to add documents to an index that doesn't exist yet, it is created automatically.

Documents are the basic building blocks of your database. Whenever you perform a search query on an index, MeiliSearch examines the data contained in your documents and returns the ones that match your query, in order of [relevancy](/guides/main_concepts/relevancy.html). Documents are represented as **[JSON objects](https://restfulapi.net/json-objects/)**.

If a document is like a book, then an index is like a bookshelf—it holds lots of related documents. Your database is like an entire library, made up of as many indexes as you desire. 
```
Trying to make it a little less overwhelming by cutting out irrelevant information and technical language.
Not sure if the analogy is helpful or just confusing. Maybe it would be better in indexes.md.
Also, this was the best description of JSON objects that I could find, but it still has a lot of JavaScript 😬 
Feel free to reject this suggestion

Reply:
It looks really complex with these blocks. Can't we put some after the code-snippets? I mean the not mandatory ones like (for example) the one about what MeiliSearch will do with your document.

And the first block seems complicated for only "you need the index name and your documents + if your index does not exist, MeiliSearch will create it". Is it mandatory
(this is only a suggestion for the first paragraph)
```suggestion
To add documents to MeiliSearch you must provide:
- [documents](/guides/main_concepts/documents.md) in the form of `JSON objects`
- the name for the [index](/guides/main_concepts/indexes.md). An index is where the documents will be stored.
If you try to add documents to an index that doesn't already exist, an index with the given name is automatically created. 
```

The sentences are not the best, this is just a suggestion, but maybe the bullets can make it lighter?